File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Policy/ClientCover.cs
using RMA.Common.Entities;

using System;
using System.Collections.Generic;

namespace RMA.Service.ClientCare.Contracts.Entities.Policy
{
    public class PolicyClientCover : AuditDetails
    {
        public int ClientId { get; set; }
        public int? PolicyId { get; set; }

        public int ProductId { get; set; }

        // public int? ProductOptionId { get; set; }
        public int? BenefitSetId { get; set; }
        public int? SkillCategoryId { get; set; }

        public DateTime StartDate { get; set; }
        public DateTime AnniversaryDate { get; set; }

        public int? NumberOfEmployees { get; set; }
        public decimal? Earnings { get; set; }
        public decimal? Premium { get; set; }
        public int? RateId { get; set; }
        public int Year { get; set; }
        public List<ClientCoverOption> ClientCoverOptions { get; set; }
    }

}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/client-cover.ts
import { ClientCoverOption} from './client-cover-option';
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class ClientCover extends BaseClass {
    policyId: number;
    productId: number;
    clientId: number;
    productOptionId: number;
    rateId: number;
    benefitSetId: number;
    numberOfEmployees: number;
    clientCoverOptions: ClientCoverOption[];
}
